src/main/python/util/utils.py

```diff
@@ -143,7 +143,7 @@ def get_tenant_info(result_loc_, org_search_, date_):
     :param date_: datetime object to pass to file path
     :return: None
     """
-    url = "{}v1/org/search".format(org_search_)
+    url = "{}/org/v1/search".format(org_search_)
```

Comment:
Is this calling the API or service endpoint directly?

Reply:
API.
Example Format: https://{{domain}}/api/org/v1/search

Reply:
This, you can use direct endpoint instead of using API. Will reduce load on the public API.